CMakeLists.txt

```diff
@@ -19,15 +19,26 @@ endif()
 # configuration files
 #######################################
 
-include(cmake/OpenEXRVersion.cmake)
-
-project(OpenEXR VERSION ${OPENEXR_VERSION} LANGUAGES C CXX)
-
-option(OPENEXR_INSTALL "Install OpenEXR libraries" ON)
-option(OPENEXR_INSTALL_TOOLS "Install OpenEXR tools" ON)
-if(OPENEXR_INSTALL_TOOLS AND NOT OPENEXR_INSTALL)
-  message(SEND_ERROR "OPENEXR_INSTALL_TOOLS requires OPENEXR_INSTALL")
+project(OpenEXR VERSION 3.0.2 LANGUAGES C CXX)
+
+set(OPENEXR_VERSION_EXTRA "" CACHE STRING "Extra version tag string for OpenEXR build")
+
+# See https://www.gnu.org/software/libtool/manual/html_node/Updating-version-info.html
+set(OPENEXR_SOVERSION 28)
+set(OPENEXR_SOAGE 0) 
+set(OPENEXR_SOREVISION 1) 
+set(OPENEXR_LIB_VERSION "${OPENEXR_SOVERSION}.${OPENEXR_SOAGE}.${OPENEXR_SOREVISION}")
```

Comment:
do you need to do the AGE/REVISION swizzle here as well?

Reply:
Yes, I submitted a separate PR #1018, when that's approved I'll cherry-pick it in, too.